README.md

```diff
@@ -30,7 +30,7 @@ staked and delegated tokens, for accounts which deploy these custom resources.
 
 ## Dependencies
 
-Go `v1.16` or higher is required to compile `flow-dps`.
+Go `v1.17` or higher is recommended to compile `flow-dps`.
```

Comment:
But go 1.16 is still required, at least, since we use the new methods from the `io` package that used to be in `ioutil`. I'm fine with saying that we recommend using 1.17 but it's still a fact that it won't compile with any go version below 1.15, and it should be noted.